test/tables.bbcode

```diff
@@ -0,0 +1,51 @@
+Simple table with caption:
+
+[table]
+[tr][th]Right[/th][th]Left[/th][th]Center[/th][th]Default[/th][/tr]
+[tr][td]12[/td][td]12[/td][td]12[/td][td]12[/td][/tr]
+[tr][td]123[/td][td]123[/td][td]123[/td][td]123[/td][/tr]
+[tr][td]1[/td][td]1[/td][td]1[/td][td]1[/td][/tr]
+[/table]
+Simple table without caption:
+
+[table]
+[tr][th]Right[/th][th]Left[/th][th]Center[/th][th]Default[/th][/tr]
+[tr][td]12[/td][td]12[/td][td]12[/td][td]12[/td][/tr]
+[tr][td]123[/td][td]123[/td][td]123[/td][td]123[/td][/tr]
+[tr][td]1[/td][td]1[/td][td]1[/td][td]1[/td][/tr]
+[/table]
+Simple table indented two spaces:
+
+[table]
+[tr][th]Right[/th][th]Left[/th][th]Center[/th][th]Default[/th][/tr]
+[tr][td]12[/td][td]12[/td][td]12[/td][td]12[/td][/tr]
+[tr][td]123[/td][td]123[/td][td]123[/td][td]123[/td][/tr]
+[tr][td]1[/td][td]1[/td][td]1[/td][td]1[/td][/tr]
+[/table]
+Multiline table with caption:
+
```

Comment:
What happens to the caption here? Shouldn't it be printed, at least as a paragraph?